package.json

```diff
@@ -2,8 +2,11 @@
   "name": "retry-axios",
   "version": "0.0.0",
   "description": "Retry HTTP requests with Axios.",
-  "main": "./build/src/index.js",
-  "types": "./build/src/index.d.ts",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.module.js",
```

Comment:
About `package.module`:

> `pkg.module` will point to a module that has ES2015 module syntax but **otherwise only syntax features that the target environments support**.
https://github.com/rollup/rollup/wiki/pkg.module

> Wait, it just means import and export – not other future JavaScript features?
> **Yes. ...**
https://github.com/rollup/rollup/wiki/pkg.module#wait-it-just-means-import-and-export--not-other-future-javascript-features